src/tests/integration/thrift/common/common.thrift

```diff
@@ -1,7 +1,7 @@
 namespace java common
 namespace js common
 
-include "shared.thrift"
+include "../shared.thrift"
```

Comment:
Does thrift somehow support includes with just the file name from a parent directory?  Need to check why this behavior changed.

Reply:
figured this out. works as it did before now.